vendor.conf

```diff
@@ -47,7 +47,7 @@ github.com/Microsoft/go-winio                       5b44b70ab3ab4d291a7c1d28afe7
 github.com/Microsoft/hcsshim                        5bc557dd210ff2caf615e6e22d398123de77fc11 # v0.8.9
 github.com/miekg/pkcs11                             210dc1e16747c5ba98a03bcbcf728c38086ea357 # v1.0.3
 github.com/mitchellh/mapstructure                   d16e9488127408e67948eb43b6d3fbb9f222da10 # v1.3.2
-github.com/moby/buildkit                            8142d66b5ebde79846b869fba30d9d30633e74aa # v0.8.1
+github.com/moby/buildkit                            d3e0a6f2d8c36dcd2d785ab436ead8bd2dc50f28 # needed for util/gitutils
```

Comment:
Should a v0.8.3 be released and referenced here instead? If so can someone help out with that?

Reply:
If this is a bug fix for v0.8.x, we need to make sure it's cherry-picked into the v0.8 branch in the buildkit repository; do you know which pull request contains the fix in the buildkit repository?

Reply:
From a quick look on GitHub, it looks like both https://github.com/moby/buildkit/pull/1782 and https://github.com/moby/buildkit/pull/1802 (which was a follow up) should be in the v0.8.2 buildkit tag;

<img width="510" alt="Screenshot 2021-04-20 at 22 35 23" src="https://user-images.githubusercontent.com/1804568/115460741-eb007780-a228-11eb-8e9a-f27d8488a61d.png">
<img width="498" alt="Screenshot 2021-04-20 at 22 35 32" src="https://user-images.githubusercontent.com/1804568/115460747-eb990e00-a228-11eb-9ef5-65f2724162cc.png">

If those are the changes needed, then I would suggest changing to;

```suggestion
github.com/moby/buildkit                            9065b18ba4633c75862befca8188de4338d9f94a # v0.8.2
```